backend/gamify/models.py

```diff
@@ -1,3 +1,12 @@
 from django.db import models
 
-# Create your models here.
+
+class Team(models.Model):
+    name = models.CharField(max_length=100, blank=False, null=False)
+    join_code = models.CharField(max_length=100, blank=False, null=False)
+
+
+class User(models.Model):
+    name = models.CharField(max_length=100, blank=False, null=False)
+    team = models.ForeignKey(Team, on_delete=models.CASCADE, blank=True, null=True)  # many-to-one
```

Comment:
We don't want to cascade here right? if a team gets deleted all the users of the team shouldn't get deleted